### DISCUSSION TITLE
¿Dónde está el repositorio del frontend de la aplicación?

### DISCUSSION THREAD

**Discussion by jonniermartinez** [Archihub-App/archihub-backend#112] [General]
Al descargar el software, pude observar el código del backend; sin embargo, cuando intenté acceder al código del frontend, me di cuenta de que todo estaba minificado. Busqué en la organización si había algún repositorio llamado "frontend" y no lo encontré.

Me pueden ayudar con el nombre del repo si saben como se llama.

**Comment by nestorandrespe** [comment]
> Al descargar el software, pude observar el código del backend; sin embargo, cuando intenté acceder al código del frontend, me di cuenta de que todo estaba minificado. Busqué en la organización si había algún repositorio llamado "frontend" y no lo encontré.
> 
> Me pueden ayudar con el nombre del repo si saben como se llama.

¡Hola! 😊 ¡Muchas gracias por tu interés en el proyecto! Efectivamente, el código del frontend de administración de este aplicativo no es público. Sin embargo, eso no significa que no puedas crear nuevas interfaces para interactuar con el contenido del sistema. 🚀

Estamos trabajando en varias interfaces abiertas que podrás usar como ejemplo para integrarlas en WordPress, proyectos en PHP, e incluso algunos componentes estarán disponibles como librerías en npm. Además de una documentación que detalle el funcionamiento del backend. El objetivo es que cualquiera pueda apropiarse del contenido que se genere en las instalaciones de la herramienta, impulsar la innovación y el reconocimiento al trabajo detrás de proyectos abiertos.

Aún nos queda mucho por avanzar en Colombia en este último aspecto, pero con iniciativas como esta, esperamos contribuir a ese cambio cultural tan necesario. 💪🌱

¡Gracias de nuevo por tu apoyo! ✨
